packages/shared/src/dto/workflows/workflow-commons-fields.ts

```diff
@@ -64,6 +64,10 @@ export class WorkflowCommonsFields {
   @IsDefined()
   name: string;
 
+  @IsString()
+  @IsDefined()
+  identifier: string;
```

Comment:
Ditto about introducing yet another identifier field on the workflow collection.

Reply:
following the new Workflow Name DX Guide i already updated to workflowId